src/mbgl/renderer/image_manager.hpp

```diff
@@ -22,21 +15,17 @@ namespace gfx {
 class UploadPass;
 } // namespace gfx
 
+class ImageManagerObserver;
 class ImageRequestor;
 
-/*
-    ImageManager does two things:
-
-        1. Tracks requests for icon images from tile workers and sends responses when the requests are fulfilled.
-        2. Builds a texture atlas for pattern images.
-
-    These are disparate responsibilities and should eventually be handled by different classes. When we implement
-    data-driven support for `*-pattern`, we'll likely use per-bucket pattern atlases, and that would be a good time
```

Comment:
This comment is removed and the original intention was per bucket pattern atlas. Is it the same as implemented in this patch? IMHO, good to put a comment, or just a link to orchestration design paper about the intention would suffice...

Reply:
This comment is outdated, as *-pattern properties were implemented long time ago as well as tile-specific atlases.